arex-schedule-web-api/src/main/java/com/arextest/schedule/model/ReplayCompareResult.java

```diff
@@ -49,6 +49,8 @@ public static ReplayCompareResult createFrom(ReplayActionCaseItem caseItem) {
         if (StringUtils.isNotBlank(caseItem.getParent().getServiceKey())) {
             newResult.setServiceName(caseItem.getParent().getServiceKey());
         }
+        newResult.setOperationName(caseItem.getParent().getOperationName());
```

Comment:
can remove the same operation in createFrom(ReplayActionCaseItem caseItem, CompareResult sdkResult)